content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst

```diff
@@ -1,95 +1,97 @@
-by going to the :menuselection:`Top Menu --> Products --> Create`, and add them later on. 
+To create a :abbr:`BoM (Bill of Materials)` from the :guilabel:`Manufacturing` module, go to
+:menuselection:`Products --> Bills of Materials`. Then, click :guilabel:`Create`. Next, specify the
+:guilabel:`Product`.
 
+.. note::
+   A :abbr:`BoM (Bill of Materials)` can also be created directly from the product form, in which
+   case the :guilabel:`Product` field is pre-filled.
 
-.. image:: bill_configuration/bom_1.png
-    :align: center
+For a standard :abbr:`BoM (Bill of Materials)`, set the :guilabel:`BoM Type` to
+:guilabel:`Manufacture this Product`. Then, click :guilabel:`Add a Line` to specify the various
+components that make up the production of the final product and their respective quantities. New
+components can be created quickly through the :abbr:`BoM (Bill of Materials)`, or can be created
+beforehand in :menuselection:`Manufacturing --> Products --> Products --> Create`. Finally, click
+:guilabel:`Save` to finish creating the :abbr:`BoM (Bill of Materials)`.
 
-.. warning::
-   The destination location should **not** be a scrap location. A scrap location is where you put
-   products that you don't need.
+.. image:: bill_configuration/bom-form.png
+   :align: center
+   :alt: Set up a Bill of Materials.
 
-Using the same BoM to describe Variants
----------------------------------------
+Specify a Bill of Materials (BoM) for a Product Variant
+-------------------------------------------------------
 
-As suggested above, you can use *BOMs* for specific *Product Variants*. 
-Once the various attributes have been configured on the product form, there 
-are two ways to configure the appropriate BoM for the respective product combination. 
+:abbr:`BoMs (Bills of Materials)` can also be assigned to specific *Product Variants*, with two
+setup options available to choose from.
 
-Either create one BoM per variant, by specifying the Product Variant
-in the dedicated field below the product name. Or use one BOM, that contains
-all of the components and for each component, indicate which variant it applies to, 
-using the *Apply on Variants* column, as shown below. 
+.. note::
+   In order to assign :abbr:`BoMs (Bills of Materials)` to product variants, the product's variant
+   attributes must already be configured on the product form.
 
-.. image:: bill_configuration/bom_2.png
-    :align: center
+The first method is to create one :abbr:`BoM (Bill of Materials)` per variant by creating a new
+:abbr:`BoM (Bill of Materials)` and specifying the :guilabel:`Product Variant`. The second method
+is to create one master :abbr:`BoM (Bill of Materials)` that contains all of the components, and
+specify which variant each component applies to using the :guilabel:`Apply on Variants` column.
 
+.. image:: bill_configuration/bom-variants.png
+   :align: center
+   :alt: Product Variants in the Bill of Materials.
 
-Adding Operations
+Set up Operations
```

Comment:
Shouldn't this be
```suggestion
Set up operations
```
?

Reply:
Same for other section titles.

Reply:
Agree. Made the changes on [eeff3ce](https://github.com/odoo/documentation/pull/2611/commits/eeff3ceb419485223e1856e94c2a3d77ccfb31b1) 👍